.github/workflows/build.yml

```diff
@@ -0,0 +1,161 @@
+name: Build cht-core and test against node versions
+
+on: [push, pull_request]
+
+env:
+  COUCH_URL: http://admin:pass@localhost:5984/medic-test
+  COUCH_NODE_NAME: nonode@nohost
+  BUILDS_SERVER: _couch/builds_testing
+  STAGING_SERVER: _couch/builds
+  UPLOAD_URL: ${{ secrets.MARKET_URL }}
+
+jobs:
+  build:
+    name: Compile the app
+    runs-on: ubuntu-18.04
+
+    steps:
+    - name: Get Docker Hub username
+      id: get-docker-hub-username
+      run: echo '::set-output name=dockerhub_username::${{ secrets.DOCKERHUB_USERNAME }}'
+    - name: Login to Docker Hub
+      uses: docker/login-action@v1
+      with:
+        username: ${{ secrets.DOCKERHUB_USERNAME }}
+        password: ${{ secrets.DOCKERHUB_TOKEN }}
+      if: steps.get-docker-hub-username.outputs.dockerhub_username
+    - name: Set UPLOAD_URL var
+      run: |
+        if [[ -z "$UPLOAD_URL" ]]; then
+          echo "UPLOAD_URL=https://staging.dev.medicmobile.org" >> $GITHUB_ENV
+          echo "BUILDS_SERVER=_couch/builds_external" >> $GITHUB_ENV
+          echo "STAGING_SERVER=_couch/builds_external" >> $GITHUB_ENV
+        fi
+    - name: Get branch name
+      uses: nelonoel/branch-name@1ea5c86cb559a8c4e623da7f188496208232e49f
+    - name: Set Travis Vars
+      run: |
+        echo "TRAVIS_BUILD_NUMBER=$GITHUB_RUN_ID" >> $GITHUB_ENV
+        echo "TRAVIS_BRANCH=$BRANCH_NAME" >> $GITHUB_ENV
+    - uses: actions/checkout@v2
+    - name: Use Node.js 12.x
+      uses: actions/setup-node@v1
+      with:
+        node-version: 12.x
+    - name: Couch Start
+      run: ./scripts/travis/couch-start
+    - name: Create logs directory
+      run: mkdir tests/logs
+    - name: npm CI
+      run: npm ci
+    - name: Grunt Install
+      run: npm install -g grunt-cli
+    - name: Configure Couch
+      run: ./scripts/travis/couch-config
+    - name: Grunt CI-Compile
+      run: |
+        node --stack_size=10000 `which grunt` ci-compile
+    - name: Publish for testing
+      run: |
+        node --stack_size=10000 `which grunt` publish-for-testing
```

Comment:
To double check, it's not necessary the default-config-tests, standard-config-tests, e2e-integration or  test-cht-default, right?

Reply:
Nope, in this version they are included in ci-compile: https://github.com/medic/cht-core/blob/7169-backport-to-310/Gruntfile.js#L1067